skimage/measure/_regionprops.py

```diff
@@ -853,7 +853,7 @@ def regionprops(label_image, intensity_image=None, cache=True,
         raise TypeError('Only 2-D and 3-D images supported.')
 
     if not np.issubdtype(label_image.dtype, np.integer):
-        if np.issubdtype(label_image.dtype, np.bool):
+        if np.issubdtype(label_image.dtype, np.bool_):
```

Comment:
```suggestion
        if label_image.dtype.kind == 'b':
```

Reply:
In CuPy we typically check the kind attribute for this sort of thing because it is very fast, but I understand if others would find it less readable. I doubt this line is critical to performance, so whichever is more maintainable is probably best.

Reply:
Good tip. 

I think it is too much microoptimization for scikit image though